lib/eztemplate/classes/eztemplatecompiler.php

```diff
@@ -1197,7 +1197,7 @@ static function processElementTransformationList( $tpl, &$node, $elements, &$pri
     static function processElementTransformationChild( $useComments, $php, $tpl, &$node,
                                                 $elementTree, $elementList, &$resourceData )
     {
-        $count = count( $elementList );
+        $count = $elementList instanceof Countable || is_array( $elementList ) ? count( $elementList ) : 0;
```

Comment:
According to doc count() returns 1 on non countable / arrays, but 0 on null. As variable is used as array beneath this is assuming 0 if not countable / array.

Reply:
Can we use https://github.com/symfony/polyfill-php73 and its `is_countable` method for this?